azure-pipelines.yml

```diff
@@ -0,0 +1,83 @@
+# File: azure-pipelines.yml
```

Comment:
Is there a reason for listing the file name in the file? E.g. does it show up in the Azure Pipelines logs and make it easier to understand where things are coming from?

Reply:
the only reason this is here AFAIK is because it was used in Azure Pipelines examples I started with, and I didn't bother removing it.  I can remove it as it doesn't really serve a purpose.